src/js/options.js

```diff
@@ -118,7 +118,7 @@ function loadOptions() {
   } else {
     // Hide WebRTC-related settings for non-supporting browsers
     $("#webRTCToggle").css({"visibility": "hidden", "height": 0});
-    $("#settingsSuffix").css({"visibility": "hidden", "height": 0});
+    $("#webrtc_warning").css({"visibility": "hidden", "height": 0});
```

Comment:
This is pretty minor, but it seems like we get extra space above `#incognito_warning` when `#webrtc_warning` is hidden. I think it's from all the `<br>`s. The fix might be to remove the `<br>`s and set appropriate padding between `<span>`s inside `#settingsSuffix`.